emukit/core/optimization/local_search_acquisition_optimizer.py

```diff
@@ -0,0 +1,110 @@
+
+        parameters = self.space.parameters
+        neighbours = []
+        current_feature = 0
+        for parameter in parameters:
+            features = parameter.round(
+                all_features[current_feature:(current_feature + parameter.dimension)]
+                .reshape(1, -1)).ravel()
+            if isinstance(parameter, CategoricalParameter):
+                if isinstance(parameter.encoding, OrdinalEncoding):
+                    left_right = np.unique([parameter.encoding.round_row(features - 1),
+                                            parameter.encoding.round_row(features + 1)])
+                    neighbours.append(left_right[left_right != features].reshape(-1, 1))
+                elif isinstance(parameter.encoding, OneHotEncoding):
+                    neighbours.append(parameter.encodings[
+                        (parameter.encodings != features).any(axis=1)])
+                else:
+                    raise NotImplementedError("Parameter encoding {} not supported."
+                                              .format(type(parameter.encoding)))
+            else:
+                raise NotImplementedError("Parameter type {} not supported."
+                                          .format(type(parameter)))
+            current_feature += parameter.dimension
+        return neighbours
+
+    def _neighbours(self, all_features: np.ndarray) -> np.ndarray:
+        assert all_features.ndim == 1, "Expected 1d array, got {}d".format(all_features.ndim)
+
+        neighbours_per_param = self._neighbours_per_parameter(all_features)
+        num_neighbours = sum(param.shape[0] for param in neighbours_per_param)
+        neighbours = np.full((num_neighbours, all_features.shape[0]), all_features)
+        current_neighbour, current_feature = 0, 0
+        for this_neighbours in neighbours_per_param:
+            next_neighbour = current_neighbour + this_neighbours.shape[0]
+            next_feature = current_feature + this_neighbours.shape[1]
+            neighbours[current_neighbour:next_neighbour,
+                       current_feature:next_feature] = this_neighbours
+            current_neighbour, current_feature = next_neighbour, next_feature
+        return neighbours
+
+    def _one_local_search(self, acquisition: Acquisition, x: np.ndarray):
+        assert x.ndim == 1, "Expected 1d array, got {}d".format(x.ndim)
+
+        incumbent_value = acquisition.evaluate(x.reshape(1, -1)).item()
+        for step in range(self.num_steps):
+            _log.info("Step {}/{}".format(step + 1, self.num_steps))
+            neighbours = self._neighbours(x)
+            acquisition_values = acquisition.evaluate(neighbours)
+            max_index = np.argmax(acquisition_values)
+            max_neighbour = neighbours[max_index]
+            max_value = acquisition_values[max_index]
+            if max_value < incumbent_value:
+                _log.info("Converged")
```

Comment:
Converged at ...

Reply:
I made these logs log more information, but made them debug logs. 
Now the local search optimizer behaves like the default one and logs to info only when it starts optimizing.